modules/ww-tritonnp/ww-tritonnp.wdl

```diff
@@ -0,0 +1,190 @@
+
+
+version 1.0
+
+# Import testdata module for automatic demo functionality
+import "https://raw.githubusercontent.com/caalo/wilds-wdl-library/refs/heads/ww-TritonNP/modules/ww-testdata/ww-testdata.wdl" as ww_testdata
+
+
+#### WORKFLOW DEFINITION ####
+
+workflow tritonnp_example {
+  meta {
+    author: "Chris Lo"
+    email: "clo2@fredhutch.org"
+    description: "TritonNP module."
+    url: "https://github.com/caalo/TritonNP"
+    outputs: {
+        final_composite: "Aggregrated outputs from all files",
+        fm_files: "Array of outputs from individal samples"
+    }
+  }
+
+  # Auto-download test data for testing purposes
+  call ww_testdata.download_tritonnp_data as download_demo_data { }
+
+  # Sample configuration
+  Array[String] sample_names = ["NA12878"]
+  Array[File] bam_paths = [download_demo_data.bam]
+  Array[File] bam_index_paths = [download_demo_data.bam_index]
+  Array[File] bias_paths = [download_demo_data.bias]
```

Comment:
Since these seem to connected it may be better to use a [`struct`](https://docs.openwdl.org/language-guide/structs.html). Might be safer than ensuring the order of all four arrays stays synced up. Then you could pass an `Array` of your struct objects as input and just scatter over that (no `[i]` indexing needed).

The [`ww-bwa`](https://github.com/caalo/wilds-wdl-library/blob/50ef7261eee30c959a40a63537353a50ad1cfa4f/modules/ww-bwa/ww-bwa.wdl) module is a good example of this